### ISSUE TITLE
Grouping form controls with FIELDSET and LEGEND, Success Criterion 1.3.1 (Info and Relationships)

### ISSUE THREAD

**Issue by asthaawasthi-laitkor** [dequelabs/axe-core#4280]
### Product

axe-core

### Question

does axe-core checks for fieldset and legend attribute ,where the rule is define ,how it is related with the WCAG 1.3.1 SC.

**Comment by WilcoFiers** [comment]
Let me check that I understand what you're asking. Do you want to know if axe-core would fail something like the following, because the `<fieldset>` does not have a `<legend>` in it?

```html
<fieldset>
  <label>Given name <input></label>
  <label>Last name <input></label>
</fieldset>
```

The answer is no, axe-core does not require that fieldsets have a legend or other accessible name. While this is often a good thing to provide, it isn't always necessary. Fieldsets are often used as a way to visually organize a form. The only time I think fieldsets require a legend is if the labels of the form fields are insufficiently descriptive. There is no good way to automatically know whether that is true or not, so axe-core doesn't test for this. It used to do so years ago, but that rule created false positives fairly regularly.

**Comment by asthaawasthi-laitkor** [comment]
Thank you for the information .@WilcoFiers
